### DISCUSSION TITLE
middleware/minio/user-guide/create/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1903] [Announcements]
# middleware/minio/user-guide/create/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/minio/user-guide/create/

**Comment by liyuerich** [comment]
安装环境检测：不是检测监控，是检测minio operator

**Comment by liyuerich** [comment]
第6步 第7步， 为什么是kakfa的图片

**Comment by ModetaNiu** [comment]
The issues mentioned above have been fixed. 
See: https://docs.daocloud.io/middleware/minio/user-guide/create/
